File: 1894-merge-strings-alternately/merge-strings-alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        n = max(len(word1),len(word2))
        ret = ""
        
        for k in range(n):
            if k<len(word1):
                ret+=word1[k]
            if k<len(word2):
                ret+=word2[k]    
        return ret

File: 1894-merge-strings-alternately/merge-strings-alternately.ts
function mergeAlternately(word1: string, word2: string): string {
    let ret = ""
    const n = Math.max(word1.length,word2.length)
    for (let k = 0; k <n; k++ ){
        if (k<word1.length) ret += word1[k]
        if (k<word2.length) ret += word2[k]
    }   
    return ret
};